test/transform/test_graph_to_simplicial_complex.py

```diff
@@ -7,6 +7,7 @@
 from toponetx.transform.graph_to_simplicial_complex import (
     graph_2_clique_complex,
     graph_2_neighbor_complex,
+    weighted_graph_to_Vietoris_Rips_complex
 )
 
```

Comment:
change weighted_graph_to_Vietoris_Rips_complex to weighted_graph_2_Vietoris_Rips_complex

to be consistent with other naming

Reply:
Changed in the new commit.